### PR TITLE
Xiaomi Miio: add EU gateway support

### PR DESCRIPTION
**Pull request by starkillerOG**
## Proposed change

Add EU gateway support documentation

## Type of change

- [ ] Spelling, grammar or other readability improvements (`current` branch).
- [ ] Adjusted missing or incorrect information in the current documentation (`current` branch).
- [ ] Added documentation for a new integration I'm adding to Home Assistant (`next` branch).
  - [ ] I've opened up a PR to add logo's and icons in [Brands repository](https://github.com/home-assistant/brands).
- [x] Added documentation for a new feature I'm adding to Home Assistant (`next` branch).
- [ ] Removed stale or deprecated documentation.

## Additional information

- Link to parent pull request in the codebase: https://github.com/home-assistant/core/pull/47955
- Link to parent pull request in the Brands repository: 
- This PR fixes or closes issue: 

## Checklist

- [x] This PR uses the correct branch, based on one of the following:
  - I made a change to the existing documentation and used the `current` branch.
  - I made a change that is related to an upcoming version of Home Assistant and used the `next` branch.
- [x] The documentation follows the Home Assistant documentation [standards][].

[standards]: https://developers.home-assistant.io/docs/documenting/standards

### CODE REVIEW

**Review by frenck — DISMISSED** [review]
✅ Approved. Can be merged as soon as the parent PR gets merged.

**Comment by frenck** [comment]
We should update these docs to clean up all the token retrieval stuff.

**Comment by starkillerOG** [comment]
@frenck I cleaned up the docs a bit, could you have a look?

File: source/_integrations/xiaomi_miio.markdown [hunk 1]
```diff
@@ -1671,3 +1488,158 @@ Set the power mode.
 |---------------------------|----------|---------------------------------------------------------------|
 | `entity_id`               |       no | Only act on a specific Xiaomi miIO switch entity.             |
 | `mode`                    |       no | Power mode, valid values are 'normal' and 'green'             |
+
+## Retrieving the Access Token
+Not recommended, please specify the cloud credentials during the config flow for easier setup.
+However when setting up a device manually the token can be retrieved in one of the following ways.
```
Comment (frenck) [inline source/_integrations/xiaomi_miio.markdown:1505]:
```suggestion
## Retrieving the Access Token

Not recommended, please specify the cloud credentials during the config flow for easier setup.
However, when setting up a device manually the token can be retrieved in one of the following ways.
```

File: source/_integrations/xiaomi_miio.markdown [hunk 2]
```diff
@@ -1671,3 +1488,158 @@ Set the power mode.
...
+The information output is:
+
+- `Device ID` - The unique identifier of the device, does not change if the device is reset.
+- `Model ID`- The model id if it could be determined, this indicates what type of device it is.
+- `Address` - The IP that the device has on the network.
+- `Token` - The token of the device or `???` if it could not be automatically determined.
+
+### Xiaomi Cloud Tokens Extractor
+
+Alternate method to get all yours devices tokens in one run. Please follow this [instruction](https://github.com/PiotrMachowski/Xiaomi-cloud-tokens-extractor).
```
Comment (frenck) [inline source/_integrations/xiaomi_miio.markdown:1656]: We should get rid of this list. Maybe we can add links to e.g., the cloud token extractor? And thinks like that?

**Review by frenck — CHANGES_REQUESTED** [review]

**Review by frenck — APPROVED** [review]

**Pull request merged**